.github/workflows/ci.yaml

```diff
@@ -115,3 +115,32 @@ jobs:
         RUSTDOCFLAGS: -D warnings
     - name: Additional per-crate checks
       run: ./tools/additional-per-crate-checks.sh ${{ matrix.runtime }}/rust-runtime/
+
+  runtime-bundle:
+    name: Produce smithy-rs runtime bundle
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
```

Comment:
seems like potentially a good use of matrix?

Reply:
In this case, I don't think so since it needs additional tar/upload steps that can't be a part of the matrix.